.github/CONTRIBUTING.md

```diff
@@ -1,65 +1,55 @@
 NewPipe contribution guidelines
 ===============================
 
-PLEASE READ THESE GUIDELINES CAREFULLY BEFORE ANY CONTRIBUTION!
-
 ## Crash reporting
 
-Do not report crashes in the GitHub issue tracker. NewPipe has an automated crash report system that will ask you to
-send a report via e-mail when a crash occurs. This contains all the data we need for debugging, and allows you to even
-add a comment to it. You'll see exactly what is sent, the system is 100% transparent.
+Report crashes in the automated crash report system of NewPipe instead of the GitHub issue tracker.
+Reports are sent via e-mail, and contain all data needed for debugging, including your comments.
+You'll see exactly what is sent, and then send it.
 
 ## Issue reporting/feature requests
 
-* Search the [existing issues](https://github.com/TeamNewPipe/NewPipe/issues) first to make sure your issue/feature
-hasn't been reported/requested before.
-* Check whether your issue/feature is already fixed/implemented.
-* Check if the issue still exists in the latest release/beta version.
-* If you are an Android/Java developer, you are always welcome to fix an issue or implement a feature yourself. PRs welcome!
-* We use English for development. Issues in other languages will be closed and ignored.
-* Please only add *one* issue at a time. Do not put multiple issues into one thread.
-* Follow the template! Issues or feature requests not matching the template might be closed.
+* **Already reported**? Browse the [existing issues](https://github.com/TeamNewPipe/NewPipe/issues) to make sure your issue/feature
+hasn't been reported/requested.
+* **Already fixed**? Check whether your issue/feature is already fixed/implemented.
+* **Still relevant**? Check if the issue still exists in the latest release/beta version.
+* **Can you fix it**? If you are an Android/Java developer, you are always welcome to fix an issue or implement a feature yourself. PRs welcome!
+* **Is it in English**? Issues in other languages will be ignored unless someone translates them.
+* **Is it one issue**?. Multiple issues require multiple reports, that can be linked.
+* **The template** Fill it out, everyone wins. Your issue has a chance of getting fixed.
 
-## Bug Fixing
-* If you want to help NewPipe to become free of bugs (this is our utopic goal for NewPipe), you can send us an email to
-<a href="mailto:tnp@newpipe.schabi.org">tnp@newpipe.schabi.org</a> to let us know that you intend to help. We'll send you further instructions. You may, on request,
-register at our [Sentry](https://sentry.schabi.org) instance (see section "Crash reporting" for more information).
+## Join the developer team
+* Help rid NewPipe of bugs by sending an e-mail saying you want to join to <a href="mailto:tnp@newpipe.schabi.org">tnp@newpipe.schabi.org</a> that says what you want to help with.
+You will get further instructions. You may be asked to register at our [Sentry](https://sentry.schabi.org) instance.
```

Comment:
We should add our IRC/Matrix channel as primary contact option.

> You will get further instructions.

I think we can remove that.

Reply:
> I think we can remove that.

Yeah, it sounds a little shady. xD

"Wait in the alley behind the apartments around midnight. You will get further instructions there."